tracpro/baseline/forms.py

```diff
@@ -0,0 +1,55 @@
+from django import forms
+
+from tracpro.polls.models import Poll
+from .models import BaselineTerm
+
+
+class BaselineTermForm(forms.ModelForm):
+    """
+    Form for Baseline Term
+    """
```

Comment:
After our talk yesterday, I'm not sure whether `region` will remain in this form. But if it does (here or elsewhere), `django-mptt` has some useful tools for forms: http://django-mptt.github.io/django-mptt/forms.html